package/etc/update-motd.d/30-services

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# set column width
+COLUMNS=3
+# colors
+green="\e[1;32m"
+red="\e[1;31m"
+undim="\e[0m"
+
+services=("appdaemon@homeassistant" "home-assistant@homeassistant" "mosquitto")
```

Comment:
I think this should only be `home-assistant@homeassistant` by default that is the only service we know is on the system.
```suggestion
services=("home-assistant@homeassistant")
```
Maybe all scripts that have a service should change til file to auto add in upon installation?